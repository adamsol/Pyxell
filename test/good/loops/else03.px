
for n in 0...5 do
    for x in 2..3 do
        if x == n do
            break
    else do
        print n
else do
    print true
