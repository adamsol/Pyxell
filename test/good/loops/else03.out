0
1
4
true
